test/examples/parameters.qmd

```diff
@@ -26,7 +26,7 @@ d
 ```
 
 ```{julia}
-print(repr(e))
+print(repr(identity.(e)))
```

Comment:
would it be better if the param arrays were more narrowly typed? Then we'd move the `identity.` into the internals

Reply:
I just thought because it's `e: ["string", "array"]` but as YAML arrays are inherently `Any` basically, it's also fine as is